src/sync/constants.js

```diff
@@ -0,0 +1,24 @@
+// time for refresh token
+export const SECONDS_BEFORE_EXPIRATION = 600;
+
+// Internal SDK events, subscribed by SyncManager and PushManager
+export const PUSH_CONNECT = 'PUSH_CONNECT';
+export const PUSH_DISCONNECT = 'PUSH_DISCONNECT';
+export const SSE_ERROR = 'SSE_ERROR';
+export const PUSH_DISABLED = 'PUSH_DISABLED';
```

Comment:
(applies to the file itself)

Keeping related constants close together makes total sense, but apply a criteria there too. Alphabetic works, priority if clear works. Anything works as long as there's a criteria that other people can understand.